src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -121,6 +131,38 @@ private function listMessages(SymfonyStyle $io, int $max)
         $io->comment('Run <comment>messenger:failed:show {id} -vv</comment> to see message details.');
     }
 
+    private function listMessagesPerClass(SymfonyStyle $io)
+    {
+        /** @var ListableReceiverInterface $receiver */
+        $receiver = $this->getReceiver();
+        $envelopes = $receiver->all();
```

Comment:
Should take "max" into account or at least throw an exception if user call this command with both `max` and `group`